broker/cmd/dejaqcli-consumer/main.go

```diff
@@ -123,12 +123,10 @@ func main() {
 	}
 
 	c.strategy = sync_consume.StrategyContinuous
-	info := "strategy: StrategyContinuous"
 	if c.StopAfterCount > 0 {
-		info = fmt.Sprintf("strategy: StrategyStopAfter %d", c.StopAfterCount)
 		c.strategy = sync_consume.StrategyStopAfter
 	}
-	logger.Info(info)
+	logger.Info(fmt.Sprintf("strategy: %v", c.strategy))
```

Comment:
infof calls sprintf so there is no need for this extra call
Infof("strategy: %s", c.strategy.String())) is enough, if it has the Stringer